components/sidebar/channel_name.jsx

```diff
@@ -64,7 +64,7 @@ export default class ChannelName extends React.PureComponent {
                 >
                     <FormattedMessage
                         id='sidebar.browseChannelDirectChannel'
-                        defaultMessage='Browse Channels and Direct Messages'
+                        defaultMessage='Browse channels and direct messages'
```

Comment:
We've always referred to channels and DMs with caps because it's a "thing" in the product. 0/5 but I prefer keeping them in caps, but will leave it up to the writing team to decide.

Reply:
Things for the win! Changed.